lib/attributes/backend/launcher.cpp

```diff
@@ -228,13 +225,11 @@ std::pair<LoopMetaData, LoopMetaData> splitTileAttr(OklLoopInfo& loopInfo, const
     // Prepare second loop
     auto secondMeta = LoopMetaData(loopInfo, r);
     secondMeta.range.start = firstMeta.var.name;
-    switch (secondMeta.condition.op) {
-        case BinOp::Le:
-            secondMeta.condition.op = BinOp::Lt;
-            break;
-        case BinOp::Ge:
-            secondMeta.condition.op = BinOp::Gt;
-            break;
+    if (secondMeta.condition.op == BinOp::Le) {
```

Comment:
Is that a correct behavior that LessEqual translates to LessThan?

Reply:
Is that okay that we skip other BinOp cases?

Reply:
This is how it was. I would imagine that there could be any kind of Binary/Unary operation for example:

```
struct MyInfo { int sizes[3] };
....
@kernel void func(..., MyInfo* ptr) {
    @outer for(int i =0 ; i <  *ptr.sizes[0]; i += *ptr.sizes[2]) {
       ...
     }
}
```
As far as I can see we just pass it as is

Reply:
> Is that a correct behavior that LessEqual translates to LessThan?
Yes. That's correct.